### ISSUE TITLE
validate all locales rather than specific locale

### ISSUE THREAD

**Issue by wingleungchoi** [bithavoc/multilang-hstore#8]
multilang :content, :required => [:en] 

The above validation will validate all locales rather than just :en.

I think line70 is the reason why it happens

In active_record_extensions.rb#L70
            # attribute presence validator
            if options[:required] == true or locale_required?(options[:required], locale)
              module_eval do
                validates_presence_of "#{attribute}_#{locale}"
              end
            end

**Comment by bithavoc** [comment]
I'm sorry, I don't understand, can you explain it differently?

**Comment by wingleungchoi** [comment]
Sorry about my poor english.

Here is my syntax
class Sample < ActiveRecord::Base

	multilang :content, :length => 500, :required => [:en]

	validates :content, :any => true
end

When i try to create instance of Sample Class,  sample1 = Sample.create(content: {en: "last 4 digital number"}), i got errors  @messages={:content_zh_HK=>["can't be blank"]}

I expect to it should just validate the presence of :en. However, it also check the presence of :zh_HK.

**Comment by bithavoc** [comment]
You're right, can You send me a pull request with a fix? Please include a test

**Comment by wingleungchoi** [comment]
I am happy to help. I will try in this weekend.
